_src/css/helpcenter/_fonts.scss

```diff
@@ -0,0 +1,25 @@
+@font-face {
+  font-family: Graphik-Medium;
+  src: url('fonts/Graphik-Medium-Web.eot');
+  src: url('fonts/Graphik-Medium-Web.eot?#iefix') format('embedded-opentype'),
+  url('fonts/Graphik-Medium-Web.woff2') format('woff2'),
+  url('fonts/Graphik-Medium-Web.woff') format('woff');
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Graphik-Regular;
+  src: url('fonts/Graphik-Regular-Web.eot');
+  src: url('fonts/Graphik-Regular-Web.eot?#iefix') format('embedded-opentype'),
+  url('fonts/Graphik-Regular-Web.woff2') format('woff2'),
+  url('fonts/Graphik-Regular-Web.woff') format('woff');
+  font-display: swap;
+}
+
+body {
+  font-family: Graphik-Medium, sans-serif;
+}
+
+p, ul {
```

Comment:
Is there a reason why we need to explicitly set the font-family for these elements? Are they overridden from the body's font family? If that's the case, are they the only elements that are overridden?

Reply:
Good question! What I was going for was setting "normal text" to Graphik-Regular, and headings, buttons, etc. to Graphik-Medium. The `p, ul` setting matches some of what the marketing site does, although it's a little more complicated than that.

It might be more correct to put `Graphik-Regular` on the body, and then override to `Graphik-Medium` for h1, h2, button, etc.